src/components/Timer.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from 'react';
+import '../styles/Timer.css'
+
+function Timer() {
+    const [currentValue, setCurrentValue] = useState(0);
+    const [intervalId, setIntervalId] = useState(false);
+
+    const resetTimer = () => {
+        clearInterval(intervalId);
+        setIntervalId(false);
+        setCurrentValue(0);
+    }
+
+    const startTimer = () => {
+        if (!intervalId) {
+            const startTime = Date.now();
+            const interval = setInterval(() => {
+                const value = (currentValue + Math.round((Date.now() - startTime) / 100) / 10);
+                setCurrentValue(value);
+            }, 100);
+            setIntervalId(interval);
+        }
+    }
+
+    const stopTimer = () => {
+        clearInterval(intervalId);
+        setIntervalId(false);
+    }
+
+    const formatTime = (numTime) => {
```

Comment:
Consider using `date-fns` to format the duration, a la https://stackoverflow.com/a/49190901

Reply:
Is this a popular library? I may revisit this one in the future.